src/components/RestaurntList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchRestaurants } from '../actions/';
+import RestaurantView from './RestaurantView';
+import './RestaurantList.css';
```

Comment:
This approach will encounter conflicts in future because simply import a simple stylesheet.
Meaning, the `.box` class in the stylesheet named `RestaurantList.css` will also affect any other `box` classes all around. Read [also](https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822)